include/aikido/planner/parabolic/ParabolicTimer.hpp

```diff
@@ -43,6 +44,28 @@ std::unique_ptr<aikido::trajectory::Spline> computeParabolicTiming(
 std::unique_ptr<aikido::trajectory::Spline> convertToSpline(
     const aikido::trajectory::Interpolated& _inputTrajectory);
 
+/// Class for performing parabolic retiming on trajectories.
+class RetimeTrajectoryPostProcessor
```

Comment:
I'd still like to rename this `ParabolicTimer` (and I think it makes more sense now that it's in this file).